CMakeLists.txt

```diff
@@ -60,15 +59,16 @@ add_library(my_vulkan_offscreen
     my_vulkan/debug_callback.cpp
     my_vulkan/helpers/standard_swap_chain.cpp
     my_vulkan/helpers/offscreen_render_target.cpp
+    my_vulkan/interop_utils.cpp
 )
 message("CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
 
-target_include_directories(
-    my_vulkan_offscreen
-    PUBLIC
-    ${OpenCV_INCLUDE_DIRS}
-    ${GLM_INCLUDE_DIRS}
-)
+#target_include_directories(
```

Comment:
why commented and not deleted?

Reply:
I do not know if remove this it will work on your side. So I left there. I can delete them.

Reply:
k, will check tomorrow